src/components/SettingsPanel.vue

```diff
@@ -166,6 +171,22 @@ export default {
       async set(value) {
         await this.changeLanguage(value);
       }
+    },
+    osKeyboardLayout: {
+      get() {
+        return this.configuratorSettings.osKeyboardLayout;
+      },
+      async set(value) {
+        await this.changeOSKeyboardLayout(value);
```

Comment:
Because this is async, I would just wrap this in a try { ... } catch (error) { console.log(error) } because it's async, it could potentially create an unhandled promise exception.

Reply:
Solved in 414986d